diagnostic.md

```diff
@@ -71,13 +74,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+relative path.  It is a relative path because we are navigating in relation to the project folder.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+It depends.  We would use a relative path if we have a local copy of the file on the desktop. I would use an absolute path if we got the image from the internet or some other location outside of our project directory.
```

Comment:
In this case we would want to use a relative path, since the image file lives inside our current working directory. As a result, an absolute path is not needed to access it.